### PR TITLE
dev-libs/json-c: fix long description formatting

### PR DESCRIPTION
**Pull request by dark**
Close double quotes, remove whitespace at EOL.

### CODE REVIEW

**Comment by dark** [comment]
Good point, thanks. I have amended the commit and renamed the PR.

**Comment by jsmolic** [comment]
You can also append `[please reassing]` to the pull request title to force PR scanning and removing of the `no signoff` label

File: dev-libs/json-c/metadata.xml [hunk 1]
```diff
@@ -18,10 +18,10 @@
...
   <longdescription lang="en">
 "A JSON implementation in C" is probably the better description, and then
-"JSON-C implements a reference counting object model that allows you to 
-easily construct JSON objects in C, output them as JSON formatted 
-strings and parse JSON formatted strings back into the C 
-representation of JSON objects.
+"JSON-C implements a reference counting object model that allows you to
+easily construct JSON objects in C, output them as JSON formatted
+strings and parse JSON formatted strings back into the C
+representation of JSON objects".
```
Comment (juippis) [inline dev-libs/json-c/metadata.xml:24]: Overall this looks more like a comment than a proper description. Can we make it clearer, or remove it altogether?

Comment (dark) [reply dev-libs/json-c/metadata.xml:24]: This message is taken almost verbatim from https://github.com/json-c/json-c. But, yes, it reads a bit weird. I will adjust it a little.

**Pull request closed without merging**